app/assets/stylesheets/bands_index.css.scss

```diff
@@ -0,0 +1,9 @@
+.bands {
+  width: 60%;
+  background: {
+    color: #170563;
```

Comment:
Color literals like `#170563` should only be used in variable declarations; they should be referred to via variable everywhere else.